src/reactviews/pages/ConnectionDialog/azureBrowsePage.tsx

```diff
@@ -192,200 +224,225 @@ export const AzureBrowsePage = () => {
-                    selection: selectedSubscription,
-                    setSelection: (sub) => {
-                        setSelectedSubscription(sub);
-
-                        if (sub === undefined) {
-                            return;
+            {!context.state.isAzureSignedIn && (
+                <div className={styles.notSignedInContainer}>
+                    <img
+                        className={styles.icon}
+                        src={azureLogoColor()}
+                        alt={Loc.connectionDialog.signIntoAzureToBrowse}
+                    />
+                    <div>{Loc.connectionDialog.signIntoAzureToBrowse}</div>
+                    <Link
+                        className={styles.signInLink}
+                        onClick={() => {
+                            context.signIntoAzureForBrowse();
+                        }}>
+                        {Loc.azure.signIntoAzure}
+                    </Link>
+                </div>
+            )}
+            {context.state.isAzureSignedIn && (
+                <>
+                    <AzureFilterCombobox
+                        label={Loc.connectionDialog.subscriptionLabel}
+                        clearable
+                        decoration={
+                            <>
+                                <Button
+                                    icon={<Filter16Filled />}
+                                    appearance="subtle"
+                                    title={Loc.connectionDialog.filterSubscriptions}
+                                    onClick={() => {
+                                        context.filterAzureSubscriptions();
+                                    }}
+                                    size="small"
+                                />
+                                {context.state.loadingAzureSubscriptionsStatus ===
+                                ApiStatus.Loading ? (
+                                    <Spinner size="tiny" />
+                                ) : undefined}
+                            </>
                         }
+                        content={{
+                            valueList: subscriptions,
+                            value: subscriptionValue,
+                            setValue: setSubscriptionValue,
+                            selection: selectedSubscription,
+                            setSelection: (sub) => {
+                                setSelectedSubscription(sub);
 
-                        // TODO: swap out subscription ID parsing for an AzureSubscriptionInfo
-                        const openParen = sub.indexOf("(");
+                                if (sub === undefined) {
+                                    return;
+                                }
 
-                        if (openParen === -1) {
-                            return;
-                        }
+                                // TODO: swap out subscription ID parsing for an AzureSubscriptionInfo
```

Comment:
Will this get updated with this PR or a seperate PR?

Reply:
I'll delete this TODO; I think it's no longer possible without upgrading the custom SearchableCombobox component, which is out of scope for this PR.